editoast/core_client/src/mocking.rs

```diff
@@ -217,23 +228,37 @@ impl StubResponseBuilder<'_> {
     /// If none is set, `AsCoreRequest::fetch` will return an `Err(CoreError::NoResponseContent)`
     #[must_use = "call .finish() to register the stub request"]
     pub fn body<B: Into<Body>>(mut self, body: B) -> Self {
-        self.body = Some(body.into());
+        match &mut self.body {
+            Some(vec) => vec.push_back(body.into()),
+            None => {
+                let mut vec = VecDeque::new();
+                vec.push_back(body.into());
+                self.body = Some(vec);
+            }
+        }
```

Comment:
You might be able to use [`get_or_insert_default`](https://doc.rust-lang.org/std/option/enum.Option.html#method.get_or_insert_default)